src/assets/scripts/client/InputController.js

```diff
@@ -502,6 +606,31 @@ export default class InputController {
         }
     }
 
+
+    /**
+     * @for InputController
+     * @method _onKeydown
+     * @param event {jquery Event}
+     * @private
+     */
+    _onKeyup(event) {
+        let { code } = event.originalEvent;
+
+        if (code === undefined) {
```

Comment:
you should actually look for:
```js
if (typeof code === 'undefined') { }
```

or, if you want to use coercion:
```js
if (code == null) { }
```

Reply:
Copy pasta from `_onKeyDown` :grin: I'll fix both in that case...